game/scripts/npc/abilities/nevermore_necromastery.txt

```diff
@@ -42,7 +42,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "necromastery_max_souls_scepter"                  "16 19 22 25 25 25" //OAA
+        "necromastery_max_souls_scepter"                  "16 19 22 25 26 27" //OAA
```

Comment:
shouldn't it be 27/29?

Reply:
I toned it down on purpose.

Reply:
ye but now the scaling is all weird since scepter is suppose to increase souls by +5, than it becomes +4 and than +3 at oaa levels